k8s/namespaces/jenkins/patches/cftptl/cluster-00/jenkins.yaml

```diff
@@ -130,7 +130,7 @@ spec:
                     title: Divorce Dashboard
                 - buildMonitor:
                     includeRegex: >-
-                      ^HMCTS_.*NFDIV.*\/(?!nfdiv-ccd-definitions|nfdiv-case-orchestration-service|nfdiv-case-maintenance-service)\/master
+                      ^HMCTS_.*NFDIV.*\/(?!nfdiv-ccd-definitions|nfdiv-case-orchestration-service|nfdiv-case-maintenance-service).+\/master
```

Comment:
what's the point of these exclusions? would it be better to simplify it?

Reply:
I see it's https://github.com/hmcts/cnp-flux-config/pull/8582

there's a better way I think

```
  - buildMonitor:
      includeRegex: "^HMCTS_.*NFDIV.*\/(nfdiv-.*)\/master"
      jobFilters:
      - statusFilter:
          statusFilter: true
```